src/Variable.cpp

```diff
@@ -616,43 +647,39 @@ vector<string>
 Variable::deputy_annotation(void) const
```

Comment:
I don't feel the need to support Deputy in Csmith anymore. I will remove it soon.